etc/openqa/openqa.ini

```diff
@@ -117,6 +117,10 @@
 #git_auto_clone = yes
 ## Experimental - Ensure a git update of all test code and needles
 #git_auto_update = no
+# whether openQA should attempt to display needles of the correct version in the web UI
```

Comment:
I have a feeling that the comment is a bit _cryptic_ from users' side. What does it mean `attempt`? Also `correct version` I think is not very explanatory. I am talking from the POV of someone who do not know much about the openQA or a new user.

Reply:
I don't think we'll be able to explain this feature within the config file. I think this comment is good enough for people trying to enable this feature after reading documentation about it elsewhere. So we could document this feature as part of the normal openQA documentation. However, I'd be fine keeping this as an almost undocumented "expert feature" for now.

Reply:
Yeah, maybe we can still change the word "correct". Who wouldn't want "correct versions"? :)

Reply:
How about

```suggestion
# whether openQA should display needles from different branches in the web UI
```

Reply:
This is not about the branch that was used (branches are moving targets) but really about the exact version that was used (not a moving target). This lookup will probably also be best effort. So I find the version of this comment as it is right now much better than the suggested change.

> Who wouldn't want "correct versions"? :)

Someone who doesn't want to pay the price for it. We could state the disadvantages here, especially that the feature is still experimental.

Reply:
So rather say "exact version" and state the drawbacks